catalog-info.yaml

```diff
@@ -213,6 +213,62 @@ spec:
 # SECTION END: DRA pipelines
 # ***********************************
 
+# ***********************************
+# SECTION START: Pull requests
+# ***********************************
+
+---
+# yaml-language-server: $schema=https://gist.githubusercontent.com/elasticmachine/988b80dae436cafea07d9a4a460a011d/raw/e57ee3bed7a6f73077a3f55a38e76e40ec87a7cf/rre.schema.json
+apiVersion: backstage.io/v1alpha1
+kind: Resource
+metadata:
+  name: logstash-pull-request-pipeline
+  description: 'Logstash Pull Request pipeline'
+  links:
+    - title: 'Logstash Pull Request pipeline'
+      url: https://buildkite.com/elastic/logstash-pull-request-pipeline
+spec:
+  type: buildkite-pipeline
+  owner: group:logstash
+  system: buildkite
+  implementation:
+    apiVersion: buildkite.elastic.dev/v1
+    kind: Pipeline
+    metadata:
+      name: logstash-pull-request-pipeline
+      description: ':logstash: Testing for Logstash Pull Requests'
+    spec:
+      pipeline_file: ".buildkite/pull_request_pipeline.yml"
+      branch_configuration: "main 7.* 8.* v7.* v8.* feature/.*"
```

Comment:
I copied/pasted this from https://github.com/elastic/elastic-agent/blob/2f18b1e6f31c1768591e9a301d52343fd96d1baa/catalog-info.yaml#L40 but I am unsure if it's needed for this use case.

Reply:
We discussed this with dimitrios, since this pipeline is only intended to be executed only for pull request, we might not need to define branch configurations. 
The elastic-agent repo is running the same pipeline for PR and main branches.

Reply:
Addressed in 19e3986beb595dc877fab375b178af172d9f9b3a